### PR TITLE
Set versions of packages

### LINKED ISSUE #74: `@rev-robotics/expansion-hub` accepts any version of the `@rev-robotics/rhsplib` dependency
The `@rev-robotics/expansion-hub` `package.json` file will be used to figure out which version of `@rev-robotics/rhsplib` to download in projects that depend on this library, so we need to specify a more specific version.

### PR DESCRIPTION
**Pull request by LandryNorris**
Fixes #74 

Sets rhsplib to 2.0.0 (es modules are a breaking change) and other modules to 1.0.0

### CODE REVIEW

File: packages/expansion-hub/package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
     "name": "@rev-robotics/expansion-hub",
-    "version": "0.1.0",
+    "version": "1.0.0",
     "description": "High level library for REV expansion hub",
     "main": "dist/index.js",
     "type": "module",
     "dependencies": {
         "serialport": "^10.5.0",
-        "@rev-robotics/rhsplib": "*"
+        "@rev-robotics/rhsplib": "2.0.0"
```
Comment (NoahAndrews) [inline packages/expansion-hub/package.json:9]:
```suggestion
        "@rev-robotics/rhsplib": "^2.0.0"
```

**Review by NoahAndrews — CHANGES_REQUESTED** [review]
We should also add a `CHANGELOG.md` file

**Comment by LandryNorris** [comment]
Added changelogs. We should change the [Unreleased] headers to the proper versions when we release.

File: packages/rhsplib/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## [Unreleased]
+
+#### Changed
+
+- Convert from commonjs to es modules
```
Comment (NoahAndrews) [inline packages/rhsplib/CHANGELOG.md:7]:
```suggestion
- Convert from commonjs to ES modules
```

**Review by NoahAndrews — APPROVED** [review]
Approving with one tiny comment

**Pull request merged**